pages/common/mosh.md

```diff
@@ -0,0 +1,33 @@
+# mosh
+
+> Mobile Shell (`mosh`) is remote terminal application that allows roaming.
+> `mosh` is a more robust and responsive replacement for SSH.
+> Visit <https://mosh.org> for more information.
+
+- Typical usage:
+
+`mosh {{username}}@{{remote_host}}`
+
+- Usage when `mosh-server` binary is outside standard path:
+
+`mosh --server={{/path/to/bin/}}mosh-server {{remote_host}}`
+
+- Connect to a remote server with a specific identity (private key):
+
+`mosh --ssh="ssh -i {{/path/to/key_file}}" {{username}}@{{remote_host}}`
+
+- Connect to a remote server using a specific port:
+
+`mosh --ssh="ssh -p {{2222}}" {{username}}@{{remote_host}}`
+
+- Run a command on a remote server:
+
+`mosh {{remote_host}} -- {{command -with -flags}}`
+
+- Select Mosh UDP port (useful when `{{remote_host}}` is behind a NAT):
+
+`mosh -p {{124}} {{username}}@{{remote_host}}`
+
+- Disable instant echo:
```

Comment:
`mosh` operates in a manner slightly different from SSH: SSH normally waits for the server to respond before it shows you what you've typed.

But `mosh` instantly echos back your typing, and does it in an adaptive manner, so as to compensate for any network issues (slowness, etc.). This makes the interface more responsive, even on high latency connections.

I've removed this example command, since the default setting (of echoing immediately) works best in most cases.